src/util.js

```diff
@@ -1767,12 +1771,12 @@ export const initialState = () => {
     contextChildren: {
       [encodeItems([ROOT_TOKEN])]: []
     },
-    lastUpdated: localStorage.lastUpdated,
+    lastUpdated: Date.now(),
```

Comment:
Actually, we want `lastUpdated` to be preserved from the local store. `initialState` does not count as an update.